docs/usage/golang.md

```diff
@@ -7,6 +7,8 @@ description: Go modules support in Renovate
 
 Renovate supports upgrading dependencies in `go.mod` files and their accompanying `go.sum` checksums.
 
+For self-hosted users, `GOPROXY`, `GONOPROXY`, `GOPRIVATE` and `GOINSECURE` environment variables are supported ([reference](https://go.dev/ref/mod#module-proxy)).
```

Comment:
```suggestion
If you're self-hosting Renovate, you may use these environment variables:

- `GOPROXY`
- `GONOPROXY`
- `GOPRIVATE`
- `GOINSECURE`

To learn what these variables do, read the [Go Modules Reference about the`GOPROXY` protocol](https://go.dev/ref/mod#module-proxy).
```

How about this?

- Put the reader into the text by saying "If you're self-hosting Renovate"
- Put the environment variables into a list
- Explain how to learn what these variables do (I'm assuming they're properly explained in the linked docs)
- Using a more descriptive link name